tests/pytorch/mock_sparse2/test_sparse_matrix.py

```diff
@@ -89,3 +89,30 @@ def test_create_from_csc(dense_dim, indptr, indices, shape):
     assert torch.allclose(mat_indptr, indptr)
     assert torch.allclose(mat_indices, indices)
     assert torch.allclose(mat_val, val)
+
+@pytest.mark.parametrize("dense_dim", [None, 4])
+@pytest.mark.parametrize("indptr", [[0, 0, 1, 4], (0, 1, 2, 4)])
+@pytest.mark.parametrize("indices", [(0, 1, 2, 3), (1, 4, 3, 2)])
+@pytest.mark.parametrize("shape", [None, (3, 5)])
+def test_csr_to_coo(dense_dim, indptr, indices, shape):
```

Comment:
Could you add tests on GPU too? For now, you could use the `ctx()` object from the test backend to get the current device being tested.

Reply:
Added